ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/DetailsPageHeader.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {
+    Flex,
+    FlexItem,
+    Label,
+    LabelGroup,
+    LabelProps,
+    Skeleton,
+    Text,
+    Title,
+} from '@patternfly/react-core';
+
+export type PageHeaderLabel = {
+    text: string;
+    icon: React.ReactNode;
+    color: LabelProps['color'];
+};
+
+export type DetailsPageHeaderProps = {
+    isLoading: boolean;
+    name: string;
+    labels?: PageHeaderLabel[];
+    summary?: string;
+    nameScreenReaderText: string;
+    metadataScreenReaderText: string;
+};
+
+const MAX_NUM_LABELS = 7;
+
+function DetailsPageHeader({
```

Comment:
Since the details page's page header will follow the same pattern for the Check Details, Cluster Details, and other details pages, I decided to make this reusable component. I will consult with the rest of the UI team to see if this is possibly something we want to reuse between features. If so, we can think about how to improve the logic here